webviz_subsurface/_datainput/pvt_data.py

```diff
@@ -159,13 +185,20 @@ def init_to_pvt_data_frame(kwargs: Any) -> pd.DataFrame:
         column_keyword = []
 
         if oil:
+            if len(oil.tables()) > 0 and isinstance(oil.tables()[0], LiveOil):
+                keyword = "PVTO"
+            else:
+                keyword = "PVDO"
```

Comment:
Might be overkill. But could be a way to catch regression bugs when making changes later?
```suggestion
            if len(oil.tables()) > 0 and isinstance(oil.tables()[0], LiveOil):
                keyword = "PVTO"
            elif len(oil.tables()) > 0 and isinstance(oil.tables()[0], DeadOil):
                keyword = "PVDO"
            else:
                raise SomeSuitableError
```